internal/cli/atlas/backup/compliancepolicy/compliancepolicy.go

```diff
@@ -37,6 +37,7 @@ func Builder() *cobra.Command {
 	cmd.AddCommand(
 		DescribeBuilder(),
 		CopyProtectionBuilder(),
+		SetupBuilder(),
```

Comment:
can you make this the first command? they print in the order you define them here when doing help

Reply:
Done